src/status.ts

```diff
@@ -37,7 +39,8 @@ export class MirrordStatus {
         const activeConfig = MirrordConfigManager.getInstance().activeConfig();
         if (activeConfig) {
             statusBar.tooltip.appendMarkdown(`\n\n[Active config: ${vscode.workspace.asRelativePath(activeConfig)}](${activeConfig})`);
-        }
+        }               
+        statusBar.tooltip.appendMarkdown(`\n\n[Auto-update: ${autoUpdate ? 'Enabled' : 'Disabled'}](command:${MirrordStatus.autoUpdateCommandId})`);        
```

Comment:
We don't want this in the menu, or as a command - it should only be available in the Extension Settings.

Reply:
oh okay, do you mean something like

```
mirrord.autoUpdate": true
```
in workspace settings.json?

Reply:
Yeah, see how binaryPath is implemented.